tests/src/main/java/testcases/T075_scope_factory_unused_dependencies/Scope.java

```diff
@@ -13,26 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package motif.errormessage
+package testcases.T075_scope_factory_unused_dependencies;
 
-import motif.core.UnusedDependencyError
+import motif.Creatable;
 
-internal class UnusedDependencyHandler(private val error: UnusedDependencyError) : ErrorHandler {
+@motif.Scope
+public interface Scope extends Creatable<Scope.Dependencies> {
```

Comment:
For some reason Git thinks this is a modification 🤔 
But actually, I removed `UnusedDependencyHandler` and added `Scope` in testcases